packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -216,6 +229,32 @@ export function IdentityHeader (props: Props) {
     );
   };
 
+  const isValidUrl = (apiUrl: string): boolean => {
```

Comment:
This function can go outside of `export function IdentityHeader`, since it's a pure function. We can avoid a re-creation on render